examples/serenity/hello_world/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+use serenity::async_trait;
+use serenity::model::channel::Message;
+use serenity::model::gateway::Ready;
+use serenity::prelude::*;
+use std::env;
+
+struct Bot;
+
+#[async_trait]
+impl EventHandler for Bot {
+    async fn message(&self, ctx: Context, msg: Message) {
+        if msg.content == "!hello" {
+            if let Err(why) = msg.channel_id.say(&ctx.http, "world!").await {
+                println!("Error sending message: {:?}", why);
```

Comment:
Maybe all the `println!` in the examples should be normal log `info!` or `error!` instead? This way they will appear in the runtime logs